src/music_module/spec/genre_spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative '../genre'
+
+describe Genre do
+  it('should test for instance variable') do
+    genre = Genre.new('John')
+    expect(genre.name).to eql('John')
+  end
+
+  it('should return an array') do
+    genre = Genre.new('James')
+    expect(genre.items).to be_a Array
+  end
+end
```

Comment:
Genre also has an `add_item` method. Kindly, add a test for this method as well.

https://github.com/Fayob/catalog_of_my_things/blob/49108f2063c9abc4b095d872a634300058cabb66/src/music_module/genre.rb#L10-L13